tests/db/model/collector/update.js

```diff
@@ -41,11 +42,69 @@ describe('tests/db/model/collector/update.js >', () => {
 
   afterEach(u.forceDelete);
 
-  it('Update status', (done) => {
-    expect(collectorDb.status).to.be.equal('Stopped'); // before
+  it('Update status [Stopped --> Running] OK', (done) => {
+    expect(collectorDb.status).to.be.equal('Stopped');
+    collectorDb.update({ status: 'Running' })
+    .then((obj) => {
+      expect(obj.status).to.be.equal('Running');
+      done();
+    })
+    .catch(done);
+  });
+
+  it('Update status [Stopped --> Paused] invalid', (done) => {
+    expect(collectorDb.status).to.be.equal('Stopped');
```

Comment:
I thought I would but I can see a use case for wanting to just stop a paused job.